content/en/docs/setup/learning-environment/_index.md

```diff
@@ -31,5 +27,20 @@ Kubernetes, or for daily development work.
 You can follow the official
 [Get Started!](https://minikube.sigs.k8s.io/docs/start/) guide if your focus is
 on getting the tool installed.
--->
+
+<a class="btn btn-primary" href="https://minikube.sigs.k8s.io/docs/start/" role="button" aria-label="View minikube Get Started! Guide">View minikube Get Started! Guide</a>
+
+Once you have `minikube` working, you can use it to
+[run a sample application](/docs/tutorials/hello-minikube/).
+
+
+## {{% heading "whatsnext" %}}
+
+- Make sure to [install `kubectl`](/docs/tasks/tools/#kubectl) as well.
+
+- If you still can't decide, pick `minikube` and start with [Hello Minikube](/docs/tutorials/hello-minikube/). `kubeadm` installs are a lot to cover for a beginner.
```

Comment:
This will be odd for readers because they've seen kind and minikube, and now we're mentioning kubeadm for the first time.

Reply:
Removed the mention!